src/store.rs

```diff
@@ -136,6 +138,19 @@ impl CertStore {
         unsafe { self.do_find(CERT_FIND_HASH, &hash_blob as *const _ as _) }
     }
 
+    /// Find list of certificates matching the SHA256 hash
+    pub fn find_by_sha256<D>(&self, hash: D) -> Result<Vec<CertContext>>
+    where
+        D: AsRef<[u8]>,
+    {
+        let hash_blob = CRYPT_INTEGER_BLOB {
+            cbData: hash.as_ref().len() as u32,
+            pbData: hash.as_ref().as_ptr() as _,
+        };
+        unsafe { self.cert_find_by_sha256(CERT_FIND_ANY, &hash_blob as *const _ as _) }
```

Comment:
>CERT_FIND_ANY [](http://example.com/codeflow?start=42&length=13)

No need for this parameter.

you want to rename to something like:
do_find_by_sha256_property

Reply:
done